### DISCUSSION TITLE
Add collection sync status indicator to project dashboard

### DISCUSSION THREAD

**Discussion by djjazzyjew** [Kong/insomnia#6637] [Ideas]
It is very helpful to see the sync status of a collection from inside the collection but it would be nice to have a way to tell which collections are in sync with the team and which ones have local changes from the project dashboard itself (see screenshot)

<img width="1558" alt="image" src="https://github.com/Kong/insomnia/assets/79609915/7bf3d669-b501-4a56-960c-26d1792db36e">

This would allow a developer to make sure their collections are in sync without having to drill down into the collections themselves.

**Comment by subnetmarco** [comment]
Yes, great idea. + @gatzjames @marckong for visibility.
